library/CodeMonkeysRu/GCM/Message.php

```diff
@@ -97,6 +97,21 @@ class Message
     private $contentAvailable = true;
 
     /**
+     * Sets the priority of the message. Valid values are "normal" and "high." On iOS, these
+     * correspond to APNs priority 5 and 10. By default, messages are sent with normal
+     * priority. Normal priority optimizes the client app's battery consumption, and should
+     * be used unless immediate delivery is required. For messages with normal priority, the
+     * app may receive the message with unspecified delay.
+     * When a message is sent with high priority, it is sent immediately, and the app can wake
+     * a sleeping device and open a network connection to your server.
+     *
+     * Optional.
+     *
+     * @var string
+     */
+    private $priority = 'normal';
```

Comment:
Let's move those string constants to explicit constants of this class.
const PRIORITY_NORMAL, etc